src/kOS/Sound/SawtoothSoundWave.cs

```diff
@@ -14,9 +14,6 @@ public SawtoothSoundWave(): base()
                 
         public override float SampleFunction(float t)
         {
-            // if (t >= 0.5f)   //eraseme
-            //    t -= 0.5f;    //eraseme
-            // return 2*t - 1;  //eraseme
```

Comment:
These aren't related to this PR, but I noticed them when I did a search for all my "// eraseme" comments to remove them at the end of my testing.  They should have been removed a while ago.